commands/orchestrationcommands/stackcommands/adopt.go

```diff
@@ -0,0 +1,159 @@
+package stackcommands
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/rackspace/rack/commandoptions"
+	"github.com/rackspace/rack/handler"
+	"github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+	osStacks "github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/openstack/orchestration/v1/stacks"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/orchestration/v1/stacks"
+	"github.com/rackspace/rack/util"
+)
+
+var adopt = cli.Command{
+	Name:        "adopt",
+	Usage:       util.Usage(commandPrefix, "adopt", "--name <stackName> --adopt-file <adoptFile>"),
```

Comment:
The docs say the" data returned by abandon stack could be provided as adopt_stack_data".  Maybe we should have a `--stdin` flag that takes `abandoned-stack` (or a more appropriate name) as its value so users can pipe from one to the other:
```
rack orchestration stack abandon --id myStackID | rack orchestration stack adopt --stdin abandoned-stack
```

@pratikmallya I'm not sure if that's a valid use-case; you'd know better than me.